### ISSUE TITLE
Convert system tests to vsphere and hosted on jenkins.puppetlabs.com

### ISSUE THREAD

**Issue by kbarber** [puppetlabs/puppetlabs-puppetdb#84]
Currently these tests are using vagrant/virtualbox, and they are pretty slow and running on a custom host. I want to be able to convert these to use the PL vsphere infrastructure.

Preferably we should solve this with the beaker bridge.

**Comment by bschonec** [comment]
@h0tw1r3, I'm gonna bet that the tests have been converted to Github workflows and this issue can be closed.


```
commit 3aaf17280d38a6dd02a45d5eb57e779971584756
Author: Jeffrey Clark <h0tw1r3@users.noreply.github.com>
Date:   Wed Feb 14 23:00:17 2024 -0600

    (maint) add release workflows

commit fe396dcc6a6bce6d744c8dc9da7bc99e1c64389a
Author: Jeffrey Clark <h0tw1r3@users.noreply.github.com>
Date:   Sun Feb 4 10:41:39 2024 -0600

    add reference validation to ci

commit 3c3a8cc43c8d81d8dc0b3431dcadeac7921a7adf
Author: Jeffrey Clark <h0tw1r3@users.noreply.github.com>
Date:   Fri Feb 2 00:53:14 2024 -0600

    remove timeouts

commit aeaa7c903c53c2ff9a4f009340fec801e86e2626
Author: Jeffrey Clark <h0tw1r3@users.noreply.github.com>
Date:   Wed Jan 31 09:21:45 2024 -0600

    update acceptance testing and add workflow

commit a8ab514eceed612b6dbc325e3e523b9d4ba7459e
Author: Jeffrey Clark <h0tw1r3@users.noreply.github.com>
Date:   Mon Jan 29 13:58:47 2024 -0600

    add github workflows for unit spec tests
nfiiseed@nfiv-schon-01d:~/a/puppetlabs-puppetdb$ 
```

**Comment by h0tw1r3** [comment]
Confirmed, fixed in #387
